hack/arktos-worker-up.sh

```diff
@@ -17,27 +17,90 @@
 set -e
 die() { echo "$*" 1>&2 ; exit 1; }
 
-if [[ ! -n "${KUBELET_IP}" ]]; then
-    die "KUBELET_IP env var not set"
+echo "DBG: Flannel CNI plugin will be installed AFTER cluster is up"
+[ "${CNIPLUGIN}" == "flannel" ] && ARKTOS_NO_CNI_PREINSTALLED="y"
+
+OPT_CNI_BIN=/opt/cni/bin
+ETC_CNI_NET_DIR=/etc/cni/net.d
```

Comment:
looks like same value, different var name in different script, suggest to use same var name. 
in [arktos-cni.rc](https://github.com/CentaurusInfra/arktos/pull/1382/files#diff-64c7b42f9b8968b0eaba4bc41373285edf52e57c6d896f0e158969321407f8a1):
```
 cni_conf_dir=${CNI_CONF_DIR:-"/etc/cni/net.d"}
cni_bin_dir=${CNI_BIN_DIR:-"/opt/cni/bin"},
```

Reply:
Thanks for your catch-up. use same name defined in hack/arktos-cni.rc.

cni_conf_dir=${CNI_CONF_DIR:-"/etc/cni/net.d"} 
cni_bin_dir=${CNI_BIN_DIR:-"/opt/cni/bin"}